### ISSUE TITLE
“DisplayManager-1.0.2.pkg” can’t be opened because Apple cannot check it for malicious software.

### ISSUE THREAD

**Issue by tjluoma** [univ-of-utah-marriott-library-apple/display_manager#13]
When I try to open the .pkg I get this error:

<img width="532" alt="Text of screenshot is below:" src="https://user-images.githubusercontent.com/26965/61004719-9da56580-a334-11e9-8857-859c853cc558.png">

“DisplayManager-1.0.2.pkg” can’t be opened because Apple cannot check it for malicious software.

This software needs to be updated. Contact the developer for more information.

Safari downloaded this file today at 5:00 PM from github-production-release-asset-2e65be.s3.amazonaws.com.

```
ProductName:	Mac OS X
ProductVersion:	10.14.5
BuildVersion:	18F132
```

**Comment by tjluoma** [comment]
I was able to work around this by doing `xattr -d com.apple.quarantine DisplayManager-1.0.2.pkg` but that's obviously not ideal

**Comment by uurazzle** [comment]
> When I try to open the .pkg I get this error:
> 
> <img alt="Text of screenshot is below:" width="532" src="https://user-images.githubusercontent.com/26965/61004719-9da56580-a334-11e9-8857-859c853cc558.png">
> 
> “DisplayManager-1.0.2.pkg” can’t be opened because Apple cannot check it for malicious software.
> 
> This software needs to be updated. Contact the developer for more information.
> 
> Safari downloaded this file today at 5:00 PM from github-production-release-asset-2e65be.s3.amazonaws.com.
> 
> ```
> ProductName:	Mac OS X
> ProductVersion:	10.14.5
> BuildVersion:	18F132
> ```

Yes, this is part of macOS 10.14.5 notarization. If you distribute your command tool as part of an Installer package, it is supposed to be possible to get the whole package notarized, although Apple hasn’t detailed a workflow for doing that, nor said whether all installer packages will be required to be notarized.

We are busy with other projects, but will add it to our todo's and will fix it when we get time.
